src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -391,10 +390,9 @@ def publishChart(registry, chartName, tag) {
 
 def deployRemoteChart(namespace, chartName, chartVersion, extraCommands) {
   withKubeConfig([credentialsId: KUBE_CREDENTIALS_ID]) {
-    sh "helm repo add ffc-demo $HELM_CHART_REPO"
-    sh "helm repo update"
-    sh "helm fetch --untar ffc-demo/$chartName --version $chartVersion"
-    sh "helm upgrade --install --recreate-pods --wait --atomic $chartName --set namespace=$namespace ./$chartName $extraCommands"
+    sh "helm3 repo add ffc $HELM_CHART_REPO"
+    sh "helm3 repo update"
+    sh "helm3 upgrade --install --atomic $chartName --set namespace=$namespace ./$chartName $extraCommands"
```

Comment:
I don't think this would work without untarring the chart as there is no `./$chartName` folder without the untar.

If just the chart repo is used the upgrade command would be something like:

`sh "helm3 upgrade --install --atomic $chartName --set namespace=$namespace ffc/$chartName $extraCommands"`

I think it's neater to not have to extract the chart for access to files inside it, but would mean anything consuming this function would need to make sure the 'extraCommands' isn't relying on a values file from inside the chart.

i.e. any extra values files would need to be held in the repo from where the command is run.

Reply:
I think that's correct, it needs the `helm fetch` to put the chart into a local folder.

I also agree it would be nicer not to fetch and extract the chart locally. When setting up the ELM Apply project, I realised that all of the extra values files we had on the demo services were redundant, given that we don't use Kubernetes in local dev environments. Some of those files had no apparent use at all anymore. As a result, ELM Apply only has the default `values.yaml` and has been working fine so far.

Reply:
Also `deployChart` creates the namespace when it doesn't exist, so maybe this one should too for consistency.
Not part of this PR, but `undeployChart` deletes the namespace along with the chart, which could have disasterous consequences if used to undeploy a chart in a shared namespace, i.e. `ffc-demo`.